pkg/operator/etcdmemberscontroller/etcdmemberscontroller.go

```diff
@@ -174,6 +176,14 @@ func (c *EtcdMembersController) reportEtcdMembers() error {
 	return nil
 }
 
+func getFormattedMessage(reportedMembers []string, allMembers []*etcdserverpb.Member, reason string) string {
```

Comment:
Would be good to have a unit test exercising these cases (makes it easier to review)